src/components/SocialButtons.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { PureComponent } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faFacebook,
+  faGooglePlus,
+  faTwitter,
+  faGithub,
+} from '@fortawesome/free-brands-svg-icons';
+
+/**
+ * Social login component
+ */
+class SocialLogin extends PureComponent {
+  /**
+   * @author frank harerimana
+   * @param {*} party
+   * @returns {*} social action
+   */
+  handlerClick(party) {
+    this.action = party;
+    return `https://strikers-ah-backend-staging.herokuapp.com/api/auth/${
+      this.action
+    }`;
+  }
+
+  /**
+   * @author frank harerimana
+   * @returns {*} component
+   */
+  render() {
+    return (
+      <div className="log-btn">
+        <div className="title-login">
+          <p>Sign-in with</p>
+        </div>
+        <div className="login-button-section">
+          <div className="social-login-btn">
```

Comment:
Hello @frankhn , I would like to suggest that social login sections can be defined as standalone components and pass them here with their respective properties. for avoiding the similar blocks from ln 37 to ln 62. This will have a good impact on the code because the lines will be few.

Reply:
Thanks for your feedback @jnkindi, basically, the social login is a component it's self and is called in Login component.